getting_started/first_3d_game/06.jump_and_squash.rst

```diff
@@ -234,25 +234,25 @@ With this code, if no collisions occurred on a given frame, the loop won't run.
  .. code-tab:: gdscript GDScript
 
    func _physics_process(delta):
```

Comment:
```suggestion
    func _physics_process(delta):
```
Space missing here for indentation to be correct, otherwise the following lines will start with 5 spaces instead of 4 on the built page. Otherwise looks good to me!

Reply:
👍 updated